packages/core/js-client/src/keypair/index.ts

```diff
@@ -67,12 +67,8 @@ export class KeyPair {
         return this.key.marshal().subarray(0, 32);
     }
 
-    signBytes(data: Uint8Array): Promise<Uint8Array> {
-        return this.key.sign(data);
-    }
-
-    verify(data: Uint8Array, signature: Uint8Array): Promise<boolean> {
-        return this.key.public.verify(data, signature);
```

Comment:
Why reduce Keypair's API and instead expose internals?

If we ever decide to change underlying crypto library, it would be a breaking change now.

Reply:
Exactly what i want to do. This object should be just a simple struct with 2 properties - publicKey and privateKey.
Why? these object smart enough already by themself. Logic which i removed was just wrapping these one line calls.